wolfcrypt/src/asn.c

```diff
@@ -25971,8 +25976,10 @@ int wc_RsaKeyToDer(RsaKey* key, byte* output, word32 inLen)
                 break;
             }
         }
-
         mpSz = SetASNIntMP(keyInt, MAX_RSA_INT_SZ, tmps[i]);
+#else
+        mpSz = SetASNIntMP(keyInt, MAX_RSA_INT_SZ, NULL);
```

Comment:
`rawLen` isn't used in the `WOLFSSL_NO_MALLOC` codepath -- need to refactor so that `rawLen` isn't declared when `WOLFSSL_NO_MALLOC`, to avoid warnings.